policybot/mgrs/syncmgr/mgr.go

```diff
@@ -1188,6 +1188,104 @@ func (ss *syncState) handleTestResults() error {
 	return nil
 }
 
+func (ss *syncState) handlePostSubmitTestResults() error {
+	for _, repo := range ss.mgr.reg.Repos() {
+		r, ok := ss.mgr.reg.SingleRecord(refresher.RecordType, repo.OrgAndRepo)
+		if !ok {
+			continue
+		}
+
+		tor := r.(*refresher.TestOutputRecord)
+		g := resultgatherer.TestResultGatherer{
+			Client:           ss.mgr.blobstore,
+			BucketName:       tor.BucketName,
+			PreSubmitPrefix:  tor.PreSubmitTestPath,
+			PostSubmitPrefix: tor.PostSubmitTestPath,
+		}
+
+		scope.Debugf("Getting test results for org %s", repo.OrgLogin)
```

Comment:
probably should reflect in the log that this is a post submit test, not presubmit.  While you're at it, can you change the log level to info, and update the corresponding log in handleTestResults() as well?  Thanks!